architecture/service_map.md

```diff
@@ -14,7 +14,7 @@ This is a good place to start if you don't know where something lives, or want t
 The above diagram shows how each of the services connect.
 For more in-depth information on how Yacs operates as a whole, see [overview](architecture/overview)
 
-### [core](https://github.com/yacs-rcos/yacs/blob/master/core)
```

Comment:
I think all of these changes are in your other PR, can we remove them from here and vice versa?

Reply:
I'll open another PR with just adapter.md changes